sdk/timing-utils/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "solana-timing-utils"
+description = "Timing utilities for Solana"
+documentation = "https://docs.rs/solana-timing-utils"
```

Comment:
nit: Since the description says "std::time utility functions" I would lean towards calling this `solana-time-utils`, which would also make it even clearer that it's different from `solana-timings`. What do you think?